From foo@baz Wed Jul 29 09:06:51 PDT 2009
Date: Wed, 29 Jul 2009 09:06:51 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: fix sparse NULL pointer warnings

From: Greg Kroah-Hartman <gregkh@suse.de>

This fixes up all of the sparse warnings where NULL should be used
instead of 0.

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/Hv.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/hv/Hv.c
+++ b/drivers/staging/hv/Hv.c
@@ -231,7 +231,7 @@ HvInit (
 	int ret=0;
     int maxLeaf;
 	HV_X64_MSR_HYPERCALL_CONTENTS hypercallMsr;
-	void* virtAddr=0;
+	void *virtAddr = NULL;
 
 	DPRINT_ENTER(VMBUS);
 
@@ -426,7 +426,7 @@ HvPostMessage(
 	alignedMsg->PayloadSize = payloadSize;
 	memcpy((void*)alignedMsg->Payload, payload, payloadSize);
 
-	status = HvDoHypercall(HvCallPostMessage, alignedMsg, 0) & 0xFFFF;
+	status = HvDoHypercall(HvCallPostMessage, alignedMsg, NULL) & 0xFFFF;
 
 	kfree((void*)addr);
 
@@ -449,7 +449,7 @@ HvSignalEvent(void)
 {
 	HV_STATUS status;
 
-	status = HvDoHypercall(HvCallSignalEvent, gHvContext.SignalEventParam, 0) & 0xFFFF;
+	status = HvDoHypercall(HvCallSignalEvent, gHvContext.SignalEventParam, NULL) & 0xFFFF;
 
 	return status;
 }
